### DISCUSSION TITLE
Training Dreambooth with old poor quality images (old 35mm photos)

### DISCUSSION THREAD

**Discussion by SorenTruelsen** [TheLastBen/fast-stable-diffusion#1324] [Q&A]
Hi Guys
I have scanned a shit load of old 35mm negatives from my youth. When compared to pictures taken today they are.. well .. out of focus, grainy and low resolution. But ... I have A LOT of them. Like from just a summer vacation I have around 100 with myself on them. 

I have tried normal DB settings, but the model was not really that good, always something wrong with eyes and facial features. I then spend quite some time trying to enhance/upscale them via different sites, it helped a bit, but still .. not too happy with the result. 

Any suggestions to settings in Dreambooth, or would a Hyper Network be better ... basically, any suggestions on how to approach this?

thanks

**Comment by TheLastBen** [comment]
use 10-15 pictures that share the same style, use the V2-512, start with 1e-5 and 800 steps unet, and 1e-6 350 steps text encoder, test the model, if not enough, resume only the unet for an additional 200 steps at 8e-6 then test again, keep adding the same amount until you get the right result.

**Reply by SorenTruelsen** [reply]
Ill try - thanks

**Reply by SorenTruelsen** [reply]
This did not really give the hoped result. I am now trying with a large amount of pictures (57) and 12000 training steps at 1e-5 to at least try to get it overtrained

**Reply by metzo007** [reply]
what is you goal?  Do you want more of the old 35mm style photos or clear and modern pictures? Or do you want to re-create a younger yourself as a model?

**Comment by metzo007** [comment]
Use the model you made first and generate 100 pictures, but add "modern, Sony RX100, shoot with a Iphone 10"....to the prompt.  Also add "out of focus, grainy,low resolution" to the negativ prompt.

After that you take the 15 best pictures and make a new dreambooth run with Bens settings above. 

Furthermore, you can use Automatc1111 to enhance the faces first, using the Extras Tab and Codeformer set to 1
